apps/els_lsp/src/els_diagnostics.erl

```diff
@@ -116,6 +118,7 @@ run_diagnostic(Uri, Id) ->
                     els_diagnostics_provider:notify(Diagnostics, self())
                 end
             },
+  ok = init_diagnostic(CbModule),
```

Comment:
This would still be run every time a file is open or saved. I think we should run this once, during initialization.

Reply:
Could you help me find better place?

Reply:
As discussed via Slack, this could be implemented the `els_general_provider` when an initialization request is received or as part of the diagnostics bootstrap. I can have a deeper look at the code to see where to hook this exactly.

Reply:
Done. But moved into `els_diagnostics_provider`.